test/runner.py

```diff
@@ -397,6 +397,8 @@ def set_env(name, option_value):
   set_env('EMTEST_CORES', options.cores)
   set_env('EMTEST_FORCE64', options.force64)
 
+  set_env('LC_CTYPE', 'latin-1')
```

Comment:
Should we instead use `with_env_modify` decorator on the tests that care about?  Then we would be testing the same thing when running locally?  I guess we could do both?